Microsoft.Toolkit.Uwp.Services/Services/MicrosoftTranslator/TranslatorServiceClient.cs

```diff
@@ -0,0 +1,284 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace Microsoft.Toolkit.Uwp.Services.MicrosoftTranslator
+{
+    /// <summary>
+    /// The <strong>TranslatorServiceClient</strong> class provides methods to translate text to various supported languages.
+    /// </summary>
+    /// <remarks>
+    /// <para>To use this library, you must register Microsoft Translator on https://portal.azure.com/#create/Microsoft.CognitiveServices/apitype/TextTranslation to obtain the Subscription key.
+    /// </para>
+    /// </remarks>
+    public class TranslatorServiceClient : ITranslatorServiceClient, IDisposable
+    {
+        private const string _baseUrl = "http://api.microsofttranslator.com/v2/Http.svc/";
```

Comment:
Constants should not have a _ before them, they should be BaseUrl, or LanguagesUri, etc